third-party/discord-rpc/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+if(WIN32)
+  find_library(DISCORD_LIB NAMES discord-rpc.lib PATHS lib)
+else()
+  find_library(DISCORD_LIB NAMES libdiscord-rpc.so PATHS lib)
```

Comment:
If I change `lib` to `./lib`, it works for me. The logic for these paths is... complicated and I have no idea why this matters. https://cmake.org/cmake/help/latest/command/find_library.html